SDL2pp/Renderer.cc

```diff
@@ -197,7 +197,7 @@ Renderer& Renderer::SetDrawColor(Uint8 r, Uint8 g, Uint8 b, Uint8 a) {
 	return *this;
 }
 
-Renderer& Renderer::SetDrawColor(const Color color) {
```

Comment:
Sorry, I've misguided you. It should better be ```const Color&```. Before, I did benchmarks for ```Rect``` vs. ```const Rect&``` arguments, and the latter was better. We should do the same way here.